webcrypto/aes.go

```diff
@@ -151,3 +155,458 @@ func importAESKey(
 
 	return key, nil
 }
+
+// AesCbcParams represents the object that should be passed as the algorithm parameter
+// into `SubtleCrypto.Encrypt`, `SubtleCrypto.Decrypt`, `SubtleCrypto.WrapKey`, or
+// `SubtleCrypto.UnwrapKey`, when using the AES-CBC algorithm.
+//
+// As defined in the [specification].
+//
+// [specification]: https://www.w3.org/TR/WebCryptoAPI/#aes-cbc-params
+type AesCbcParams struct {
+	Algorithm
+
+	// Name should be set to AES-CBC.
+	Name string `json:"name"`
+
+	// Iv holds (an ArrayBuffer, a TypedArray, or a DataView) the initialization vector.
+	// Must be 16 bytes, unpredictable, and preferably cryptographically random.
+	// However, it need not be secret (for example, it may be transmitted unencrypted along with the ciphertext).
+	Iv []byte `json:"iv"`
```

Comment:
In theory I would agree with you, but in practice, the aes cipher functions expect a slice, thus I would need to convert it to a slice with the `[:]` which, I believe would lead to effectively copy the content of the array on the stack on the heap: which would be inefficient? I might have the wrong assumption though.

Also, quickly testing the solution, it seems Goja's `ExportTo` method's not fond of arrays.